feat: add push notification to iOS (#268)
HathorNetwork/hathor-wallet-mobile: README.md

First start the Metro bundler:

`npm start` or `npm run start:clean` to invalidate the cache before run.

Then build the app for iOS:

`npm run ios` to open on iOS simulator, or `npm run ios -- --simulator "iPhone 11"` to run it by name.

To list all available simulators, use `xcrun simctl list devices`.

#### Physical device

If you want to run in your physical device, you should first get its UDID by running the list of all possible devices:

```bash
xcrun xctrace list devices
```

You should see an output like this:
```text
== Devices ==
Alex’s ... (490613AE-...)
Alex’s ... (16.3.1) (3198...)

== Simulators ==
...
```

The code inside the last parenteses is your UDID. Then, deploy the app to your device with:

```bash
npm run ios -- --udid=3198...
```

> > INFO: To be able to deploy to your physical device you need to install the command `ios-deploy` first:
> > `brew install ios-deploy; zsh; ios-deploy --version`

Reviewer: That implies that the user is using `brew` and `zsh`, maybe just link the `ios-deploy` project:

```suggestion
> INFO: To be able to deploy to your physical device you can use `ios-deploy`, it can be downloaded from https://github.com/ios-control/ios-deploy:
> `ios-deploy --version`
```
